### PR TITLE
Clock desktop application added

### LINKED ISSUE #360: Title: Desktop clock application added(python)
### Explain what you want to add/change in documentation.

Desktop clock application 
Techstack-Python
library used-Time module of python
program-GsSOC2022

### PR DESCRIPTION
**Pull request by Ash-KODES**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.)

### Description

(Write your answer here.)

Fixes #360 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by Ash-KODES** [comment]
please review

**Comment by Ash-KODES** [comment]
Sir,all the changes have been made except the project name.
I dont know how to change the name of a folder,please guide me?

File: Desktop Application/Basic/Python/Desktop Clock/Readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Digital Clock Application
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Desktop Clock/Readme.md:1]: Add a good gif or photos about your project in `readme.md` file.

**Comment by Kushal997-das** [comment]
> Sir,all the changes have been made except the project name. I dont know how to change the name of a folder,please guide me?



> Sir,all the changes have been made except the project name. I dont know how to change the name of a folder,please guide me?

You can change your project name from local.

**Comment by Ash-KODES** [comment]
Finally Done Sir :) Thankyou for  guidance!

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Ash-KODES Can you please tell me what is that (.TTF) files if they are not related to your project then please delete them .

**Comment by Ash-KODES** [comment]
> @Ash-KODES Can you please tell me what is that (.TTF) files if they are not related to your project then please delete them .

Yes,They are related to project,,Its the Icons that I used for my clock's UI.

**Comment by Kushal997-das** [comment]
> > @Ash-KODES Can you please tell me what is that (.TTF) files if they are not related to your project then please delete them .
> 
> Yes,They are related to project,,Its the Icons that I used for my clock's UI.

Okey!!

**Review by Kushal997-das — APPROVED** [review]
@Ash-KODES 

LGTM 👍 

@photon149 please check !!

**Comment by photon149** [comment]
Approved from my end.
@prakhar3062 over to you.

**Review by prakhar3062 — APPROVED** [review]
okay from my end

**Review by Kushal997-das — APPROVED** [review]
@Ash-KODES Great 💯

**Pull request merged**